### PR TITLE
es: Translations - Concurrency

### PR DESCRIPTION
**Pull request by henrif75**
Part of #284

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -17365,44 +17370,51 @@ msgstr ""
...
 
 #: src/concurrency/threads.md:25
 msgid "Thread panics are independent of each other."
-msgstr ""
+msgstr "Los pánicos de los hilos son independientes entre sí."
 
 #: src/concurrency/threads.md:26
 msgid "Panics can carry a payload, which can be unpacked with `downcast_ref`."
 msgstr ""
+"Los pánicos pueden transportar una carga útil, que se puede descomprimir con "
```
Comment (deavid) [inline po/es.po:17382]:
```suggestion
"Los pánicos pueden transportar una carga útil, que se puede desempaquetar con "
```

File: po/es.po [hunk 2]
```diff
@@ -17691,134 +17734,165 @@ msgid ""
...
 #: src/concurrency/send-sync/examples.md:27
 msgid "`!Send + Sync`"
-msgstr ""
+msgstr "`!Send + Sync`"
 
 #: src/concurrency/send-sync/examples.md:29
 msgid ""
 "These types are thread-safe, but they cannot be moved to another thread:"
 msgstr ""
+"Estos tipos son seguros para los hilos, pero no se pueden mover a otro hilo:"
```
Comment (deavid) [inline po/es.po:17824]:
```suggestion
"Estos tipos son seguros para los hilos _(thread safe)_, pero no se pueden mover a otro hilo:"
```

File: po/es.po [hunk 3]
```diff
@@ -18060,13 +18175,25 @@ msgid ""
 "not eating. After an individual philosopher finishes eating, they will put "
 "down both forks."
 msgstr ""
+"Cinco filósofos cenan juntos en la misma mesa. Cada filósofo tiene su propio "
+"sitio en ella. Hay un tenedor entre cada plato. El plato que van a degustar "
+"es una especie de espaguetis que hay que comer con dos tenedores. Los "
+"filósofos solo puede n>pensar y comer alternativamente. Además, solo pueden "
```
Comment (deavid) [inline po/es.po:18181]:
```suggestion
"filósofos solo pueden pensar y comer alternativamente. Además, solo pueden "
```

File: po/es.po [hunk 4]
```diff
@@ -18464,75 +18748,107 @@ msgid ""
...
+"punteros en ese futuro siguen siendo válidos. Esto es necesario para que las "
+"referencias sigan siendo válidas después de `.await`."
 
 #: src/async/runtimes.md:3
 msgid ""
 "A _runtime_ provides support for performing operations asynchronously (a "
 "_reactor_) and is responsible for executing futures (an _executor_). Rust "
 "does not have a \"built-in\" runtime, but several options are available:"
 msgstr ""
+"Un _tiempo de ejecución_ ofrece asistencia para realizar operaciones de "
```
Comment (deavid) [inline po/es.po:18805]:
```suggestion
"Un _runtime_ ofrece asistencia para realizar operaciones de "
```

This is a different "run time" from the usual. This does not refer "at runtime" but to something called "runtime" in async rust. I don't think we have a translation for that.

File: po/es.po [hunk 5]
```diff
@@ -18853,6 +19221,11 @@ msgid ""
 "ready, the `statement` is executed with the variables in `pattern` bound to "
 "the `future`'s result."
 msgstr ""
+"Al igual que en el caso de una instrucción de coincidencia, el cuerpo de "
```
Comment (deavid) [inline po/es.po:19224]:
```suggestion
"Al igual que en el caso de una instrucción _match_, el cuerpo de "
```

**Review by deavid — CHANGES_REQUESTED** [review]

**Review by deavid — APPROVED** [review]

**Comment by deavid** [comment]
And with this, seems we are done.
I'll be OOO for real pretty soon. I'll check once I'm back.

**Comment by henrif75** [comment]
> And with this, seems we are done. I'll be OOO for real pretty soon. I'll check once I'm back.

Thank you so much @deavid !

**Pull request merged**